typewise_alert.py

```diff
@@ -12,23 +12,32 @@ def infer_breach(value, lowerLimit, upperLimit):
   return 'NORMAL'
 
 def classify_temperature_breach(coolingType, temperatureInC):
-  temperature_range=individual_temperature_range[coolingType]
-  return infer_breach(temperatureInC, temperature_range['lowerLimit'], temperature_range['upperLimit'])
+  temperature_range=coolingType_temperature_range.get(coolingType)
+  if(temperature_range!=None):
+    return infer_breach(temperatureInC, temperature_range['lowerLimit'], temperature_range['upperLimit'])
+
 
 def check_and_alert(alertTarget, batteryChar, temperatureInC):
   breachType =classify_temperature_breach(batteryChar['coolingType'], temperatureInC)
-  message_target[alertTarget](breachType)
+  if(breachType!=None):
+    #message_to_target[alertTarget](breachType)
```

Comment:
Do not leave commented code in commits. If you need it later, it will always be there in the history.